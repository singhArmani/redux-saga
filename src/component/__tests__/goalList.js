import React from 'react';
import GoalList from '../goalList';

import renderer from 'react-test-renderer';


describe('GoalList', () => {
  const testProps = {
    isFetchingGoals: false,
    goalList: ['test goal 1', 'test goal 2']
  };

  it('renders correctly', () => {
    const tree = renderer.create(<GoalList {...testProps} />).toJSON();
    expect(tree.children.length).toBe(2);
    expect(tree).toMatchSnapshot();
  });
});
