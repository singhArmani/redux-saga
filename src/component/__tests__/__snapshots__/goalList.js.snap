// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoalList renders correctly 1`] = `
<ul>
  <li>
    test goal 1
  </li>
  <li>
    test goal 2
  </li>
</ul>
`;
